blogster/app/views/comments/_comment.html.erb

```diff
@@ -1,24 +1,32 @@
-<div class="comment-cell">
-  <div class="comment-avatar">
-    <% if comment.user.image_url.to_s.empty? %>
-      <%= render 'shared/placeholders/avatar' %>
-    <% else %>
-      <%= image_tag comment.user.image_url.to_s, alt: "User avatar", class: "img-responsive" %>
-    <% end %>
-  </div>
-  
-  <div class="comment-cell-user inline-container">
-    <h4><%= comment.user.name %></h4>
-    <i><%= comment.created_at.strftime("%d/%m/%y, %H:%M:%S") %></i>
-
-    <% if user_signed_in? && comment.user.id == current_user.id %>
-      <span class="comment-actions">
-        <%= link_to "", edit_comment_path(comment), class: "fas fa-edit fa-lg comment-button" %>
-        <%= link_to "", comment, class: "fas fa-times fa-lg comment-button",
-          method: :delete, data: { confirm: "Are you sure?" } %>
-      </span>
-    <% end %>
-  </div>
-
-  <div class="comment-cell-content"><%= comment.content %></div>
+<% if (defined? link_posts) && (link_posts == true) %>
+  <h4>
+    <%= link_to comment.post.title, post_path(comment.post), class: "comment-link-post" %>
+  </h4>
+<% end %>
+
+<div class="comment-cell">
+  <div class="comment-avatar">
+    <% if comment.user.image_url.to_s.empty? %>

```

Comment:
o uso do `present?` é mais facíl de entender, acredito que fez desta maneira devido erro ao usar `empty?` tem um artigo no stackoverflow que explica um pouco
https://stackoverflow.com/questions/1654960/when-to-use-nil-blank-empty

mas para o `CarrierWave` vc pode usar `<% if comment.user.image_url.present? %>` e inverter as condições ou use o `unless`